Main/src/Coffee/Action.java

```diff
@@ -0,0 +1,186 @@
+                break;
+        }
+}
+
+    public static void orderCoffee(Scanner in){
+        System.out.println("Step 2: Waiting for the order");
+        System.out.println("Please input coffee type ");
+        System.out.println("1. Espresso");
+        System.out.println("2. Latte");
+        System.out.println("3. Cappuccino");
+        System.out.println("or print 'back' to return... ");
+        String coffeeType = in.nextLine();
+
+
+        switch(coffeeType) {
+            case "Espresso":
+                System.out.println("Please input cups quantity ");
+                int cup = in.nextInt();
+                Action.espresso(cup);
+                break;
+            case "Latte":
+                System.out.println("Please input cups quantity ");
+                cup = in.nextInt();
+                Action.latte(cup);
+                break;
+            case "Cappuccino":
+                System.out.println("Please input cups quantity ");
+                cup = in.nextInt();
+                Action.cappuccino(cup);
+                break;
+            case "back":
+                getAction();
+                in.close();
+                break;
+        }
+
+        in.close();
+    }
+
+        public static void fillMachine()
+    {  stepOne: ;
+        Scanner in = new Scanner(System.in);
+        System.out.println("Please start to fill the water tank, input water volume... ");
+        int water = in.nextInt();
+        Action.water += water;
+        System.out.println("Please start to fill the milk tank, input milk volume... ");
+        int milk = in.nextInt();
+        Action.milk += milk;
+        System.out.println("Please start to fill the coffee beans tank, input coffee volume... ");
+        int coffeeBeans = in.nextInt();
+        Action.coffeeBeans += coffeeBeans;
+        System.out.println("We starts to fill the cups tank, input cups quantity... ");
+        int dcups = in.nextInt();
+        Action.dcups += dcups;
+        in.close();
+    }
+
+    public static void pull(){
+        System.out.println("Collected " + Action.money + " dollars.");
+        Action.money = 0;
+        // break stepOne; //вернуться в степ1
+    }
+
+    public static void machineStatus(){
+        System.out.println("In Coffee Machine we have: " + Action.water + " ml of water, " + Action.milk + " ml of milk, " + Action.coffeeBeans + " gr of beans, " + Action.dcups + " cups.");
+    }
+
+    public static void espresso(int cup){
+
+        if (checkEspressoAvailability (cup) == true) {
+            Action.water -= (250 * cup);
+            Action.coffeeBeans -= (16 * cup);
+            Action.money += (4 * cup);
+            Action.dcups -= (1 * cup);
+            System.out.println("Price is 4$ for cup. Thank you for your purchase, please take your espresso.");
+        }
+    }
+    public static void latte(int cup) {
+        if (checkLatteAvailability(cup) == true) {
+            Action.water -= (350 * cup);
```

Comment:
using class field as a static or public ints not such a good idea, please change to private and add some methods which is going to modify your Access class fields

Reply:
@GayaneGulanyan not fixed

Reply:
please explain me, we did it today with object during Q&A, was it the solution?

Reply:
Yes just use getters or setters, as I show on Q&A

Reply:
Used object and it's method in line 13, 14 -> CoffeeMachineSimulatorMain.java -> CoffeeMachineSimulatorMain.java